app/views/taxonomy_projects/new.html.erb

```diff
@@ -0,0 +1,53 @@
+<h1>Create new project</h1>
+
+<div class='row'>
+  <div class='col-md-8'>
+    <%= form_for @form, url: taxonomy_projects_path do |f| %>
+      <div class="form-group">
+        <%= f.label :name, "Name", class: "control-label" %>
+
+        <div class="form-wrapper">
+          <%= f.text_field :name, class: "form-control" %>
+        </div>
+      </div>
+
+      <div class="form-group">
+        <%= f.label :csv_url, "Google Spreadsheet URL", class: "control-label" %>
+
+        <div class="form-wrapper">
+          <%= f.text_field :csv_url, class: "form-control" %>
+        </div>
+
+        <p>The spreadsheet should have a <code>content_id</code> column with the
+          content IDs of the pages. </p>
+      </div>
+
+      <%= f.submit "Save", class: "btn btn-success" %>
+    <% end %>
+  </div>
+
+  <div class='col-md-4'>
```

Comment:
These instructions come from content-tagger and have been tested with a content designer.